books/templates/user.html

```diff
@@ -3,11 +3,76 @@
 {% load humanize %}
 {% load tags %}
 
+{% block css %}
+  <style>
+    .js-favorite {
```

Comment:
why only here as opposed to in style.css in static?

Reply:
this was a random choice. i can move it to style.css as this is generic enough.